doc/radosgw/pubsub-module.rst

```diff
@@ -68,28 +84,18 @@ The oid prefix for the stored events.
 
 How many days to keep events that weren't acked.
 
-How to Configure
-~~~~~~~~~~~~~~~~
+How to Configure?
+~~~~~~~~~~~~~~~~~
 
-See `Multisite Configuration`_ for multisite configuration instructions. The pubsub sync module requires a creation of a new zone. The zone
-tier type needs to be defined as ``pubsub``:
-
-::
-
-   # radosgw-admin zone create --rgw-zonegroup={zone-group-name} \
-                                --rgw-zone={zone-name} \
-                                --endpoints={http://fqdn}[,{http://fqdn}]
-                                --tier-type=pubsub
-
-The tier configuration can be then done using the following command
+The tier configuration could be set using the following command:
 
 ::
 
    # radosgw-admin zone modify --rgw-zonegroup={zone-group-name} \
                                 --rgw-zone={zone-name} \
                                 --tier-config={key}={val}[,{key}={val}]
 
-The ``key`` in the configuration specifies the configuration variable that needs to be updated, and
+Where the ``key`` in the configuration specifies the configuration variable that needs to be updated, and
```

Comment:
I think we should be more clear about what are the keys necessary for the pubsub to work

Reply:
aren't these spelled out in `PubSub Zone Configuration Parameters` section directly above?

Reply:
i think that this is an alternative to setting the zone parameters in a conf file. these CLI commands are showing how to set the parameters via CLI - will change the headline so it is clear